modules/@apostrophecms/ai-helper-rich-text-widget/ui/apos/components/AposAiHelperTextDialog.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div
+    v-if="active"
+    v-click-outside-element="close"
+    class="apos-popover apos-ai-helper-text__dialog"
+    x-placement="bottom"
+    :class="{
+      'apos-is-triggered': active,
+      'apos-has-selection': true
+    }"
+  >
+    <AposContextMenuDialog
+      menu-placement="bottom-start"
+    >
+      <form class="apos-ai-helper-form">
+        <p>
+          {{ $t('aposAiHelper:textPromptLabel') }}
+        </p>
+        <textarea v-model="prompt" />
+        <p v-if="error">
+          An error occurred.
+        </p>
+      </form>
+      <footer class="apos-ai-helper-text__footer">
+        <AposButton
+          type="default" label="apostrophe:cancel"
+          @click="close"
+          :modifiers="formModifiers"
+        />
+        <AposButton
+          type="primary" label="aposAiHelper:generateTextAction"
+          @click="save"
+          :disabled="!prompt.length"
+          :modifiers="formModifiers"
+        />
+      </footer>
+    </AposContextMenuDialog>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AposAiHelperTextDialog',
+  emits: [ 'close', 'before-commands' ],
+  props: {
+    editor: {
+      type: Object,
+      required: true
+    },
+    active: {
+      type: Boolean,
+      required: true
+    },
+    options: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      prompt: '',
```

Comment:
Could we store the prompt somewhere?
Testing the feature, I find it would be really nice to get the previous prompt in order to sharpen the result from it.

This could be part of a future PR